scripts/generate_test_suites.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+
+parts = sys.argv[1]
+
+instrumentation_tests_path = 'instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests'
+core_tests = os.listdir(f'{instrumentation_tests_path}/core')
+ui_tests = os.listdir(f'{instrumentation_tests_path}/ui')
+
+core_part_size = int(len(core_tests) / int(parts)) + 1
+
+suites = {}
```

Comment:
Can we add a check somewhere in the end that **all** test classes appeared in some suite? I want to avoid a situation when some tests are just not executed and we don't notice it.